spec/system/home_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Welcome page', type: :system do
+  it 'can see the app title' do
+    visit unauthenticated_root_path
+    expect(page).to have_content('Expense App')
+  end
+
+  it 'can see the login button' do
+    visit unauthenticated_root_path
+    expect(page).to have_content('Log in')
+  end
+
+  it 'can see the signup link' do
+    visit unauthenticated_root_path
+    expect(page).to have_content('Sign Up')
+  end
+end
```

Comment:
- [ ] Good Job for provide integration tests for Welcome page and test cases are passing But As stated in the project requirement  **kindly make sure that you provide Integration tests for all(Home page (categories page), Transactions page) the most important components of the RoR application.**

Reply:
noted, changes has been made in code